### DISCUSSION TITLE
Any resources on how to create a plugin for helix?

### DISCUSSION THREAD

**Discussion by glyh** [helix-editor/helix#9903] [Q&A] [answered]
Hello, I see the plugin system landed in helix with a scheme implementation, which is nice, is there any resources on how to write a plugin for helix now? Thanks!

**Comment by gabydd** [comment, marked as answer]
Not sure where you heard about a plugin system landing in helix, but one has not. There is an MVP being worked on that you may have heard about here: #8675 but nothing final yet.
